openstudiocore/resources/analysis/pat_export/scripts/StandardReports/measure.rb

```diff
@@ -1,139 +1,165 @@
 require 'erb'
+require 'json'
+
```

Comment:
The files in openstudiocore/resources/analysis/pat_export/scripts seem to only be used in this unit test, https://github.com/NREL/OpenStudio/blob/bf5f68c76b79a69deaff25aae05cbb8a798b31b1/openstudiocore/src/analysis/test/Analysis_GTest.cpp.  So it is fine to update them but not critical.  We should make sure the unit tests still pass.